File: C/URI/1074.c
#include <stdio.h>
 
int main() {
 
  int n, numero;
        scanf("%d", &n);
         while(n > 0)
         {
                scanf("%d", &numero);
                if(numero%2 == 0 && numero > 0)
                {
                        printf("EVEN POSITIVE\n");
                }
                else if(numero%2 == 1 && numero > 0)
                {
                        printf("ODD POSITIVE\n");
                }
                else if(numero%2 == 0 && numero < 0)
                {
                        printf("EVEN NEGATIVE\n");
                }
                else if(numero%2 == -1 && numero < 0)
                {
                        printf("ODD NEGATIVE\n");
                }
                else if(numero == 0)
                {
                        printf("NULL\n");
                }
        n = n -1;

         }


 
    return 0;
}

File: php/URI/1074.php
<?php

$caso = trim(fgets(STDIN));

for (; $caso >0; $caso--) {
    $num = trim(fgets(STDIN));
    if($num == 0)
        echo"NULL\n";
    if($num % 2 != 0 && $num < 0)
        echo"ODD NEGATIVE\n";
    if($num % 2 == 0 && $num < 0)
        echo"EVEN NEGATIVE\n";
    if($num % 2 != 0 && $num > 0)
        echo"ODD POSITIVE\n";
    if($num % 2 == 0 && $num > 0)
        echo"EVEN POSITIVE\n";

}

?>